webui/src/pages/auth/groups/index.tsx

```diff
@@ -171,6 +175,9 @@ const GroupsContainer = () => {
                 placeholder="Group Name (e.g. 'DataTeam')"
                 actionName={"Create"}
                 validationFunction={disallowPercentSign(INVALID_GROUP_NAME_ERROR_MESSAGE)}
+                showExtraField={true}
+                extraPlaceholder="Group Description (optional)"
+                extraValidationFunction={disallowPercentSign(INVALID_GROUP_DESCRIPTION_ERROR_MESSAGE)}
```

Comment:
Please only add such protections if there is an actual broken case.  I always prefer to employ correct quotation mechanisms rather than to attempt to prevent all "bad" cases.

I would expect URI component encoding or whatever to protect percent signs.  If things break when descriptions have percent signs, then I expect them also to break in many other cases, such as descriptions with plus signs.